internal/review.go

```diff
@@ -0,0 +1,195 @@
+package internal
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"slices"
+	"strings"
+
+	"github.com/mitchellh/colorstring"
+	"github.com/spf13/cobra"
+
+	"github.com/nobl9/nobl9-go/sdk"
+)
+
+var (
+	errReviewTooManyArgs    = errors.New("'review set' command accepts only single SLO name as an argument")
+	errReviewInvalidOptions = errors.New("you must provide the SLO name as an argument")
+)
+
+type ReviewCmd struct {
+	client  *sdk.Client
+	project string
+	status  string
+	note    string
+	sloName string
+}
+
+//go:embed review_example.sh
+var reviewExample string
+
+// NewReviewCmd returns cobra command review with all its flags and subcommands.
+func (r *RootCmd) NewReviewCmd() *cobra.Command {
+	review := &ReviewCmd{}
+
+	cmd := &cobra.Command{
+		Use:   "review",
+		Short: "Manage SLO review (Enterprise Edition only)",
+		Long: `Manage SLO review.
+
+Note: This feature is only available in Enterprise Edition tier.`,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			review.client = r.GetClient()
+		},
+	}
+
+	cmd.AddCommand(review.NewSetCmd())
+
+	return cmd
+}
+
+// NewSetCmd creates the set subcommand for reviews
+func (r *ReviewCmd) NewSetCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set <SLO_NAME>",
+		Short: "Set SLO review status (Enterprise Edition only)",
+		Long: `Set SLO review status.
+
+This command allows you to update the review status of a specific SLO within a project.
+Available statuses are: reviewed, skipped, pending, overdue, notStarted.
+
+When setting status to 'reviewed' or 'skipped', you can optionally include a note
+using the --note flag to provide additional context or reasoning for the review decision.
+
+The SLO name must be provided as the first argument, and the project can be specified
+using the --project flag or will default to the configured project in your client.
+
+Note: This feature is only available in Enterprise Edition tier.`,
+		Example: reviewExample,
+		Args:    r.reviewSetArguments,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if r.project == "" {
+				r.project = r.client.Config.Project
+			}
+			return r.runSetReview(cmd, r.sloName)
+		},
+	}
+
+	cmd.Flags().StringVar(&r.status, "status", "",
+		"Review status: reviewed, skipped, pending, overdue, notStarted (required)")
```

Comment:
maybe define these statuses in a global var and use them in validation and in this message for maintainability